streampark-e2e/streampark-e2e-case/src/test/java/org/apache/streampark/e2e/pages/setting/entity/FaultAlertSetting.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.e2e.pages.setting.entity;
+
+public abstract class FaultAlertSetting {
```

Comment:
There is a `Fault Alert Type` field on the page. The naming is referred to here: 

> <img width="653" alt="截屏2024-07-20 下午8 51 56" src="https://github.com/user-attachments/assets/d51c6218-87cb-4fdb-96bf-4a3f8bc8f992">